specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-03-01/managedClusters.json

```diff
@@ -1470,6 +1470,10 @@
           "$ref": "#/definitions/AgentPoolType",
           "description": "AgentPoolType represents types of an agent pool"
         },
+        "mode": {
```

Comment:
Adding a writable field to an exiting API version is considered a breaking change. Please include this change in a new API version.

See 'New property added to response':
https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo

Reply:
Hi @ryansbenson ,  2020-03-01 api-version is not published yet. This PR is to merge to the dev branch and we will merge to master later when all of our March changes are merged. I know we are a bit late. So it is a bit confusing.